src/main/java/net/glowstone/entity/objects/GlowEnderPearl.java

```diff
@@ -0,0 +1,106 @@
+package net.glowstone.entity.objects;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.bukkit.Location;
+import org.bukkit.Material;
+import org.bukkit.Sound;
+import org.bukkit.entity.EnderPearl;
+import org.bukkit.entity.Projectile;
+import org.bukkit.projectiles.ProjectileSource;
+import org.bukkit.util.Vector;
+
+import com.flowpowered.network.Message;
+
+import net.glowstone.command.minecraft.PlaySoundCommand;
+import net.glowstone.entity.GlowEntity;
+import net.glowstone.entity.GlowPlayer;
+import net.glowstone.net.message.play.entity.EntityMetadataMessage;
+import net.glowstone.net.message.play.entity.EntityTeleportMessage;
+import net.glowstone.net.message.play.entity.EntityVelocityMessage;
+import net.glowstone.net.message.play.entity.SpawnObjectMessage;
+import net.glowstone.util.Position;
+
+public class GlowEnderPearl extends GlowEntity implements EnderPearl{
+	private ProjectileSource shooter;
+	
+	public GlowEnderPearl(Location location) {
+		super(location);
+		setDrag(0.99, false);
+		setDrag(0.99, true);
+		setGravityAccel(new Vector(0,-0.06,0));
+	}
+
+	@Override
+	public boolean doesBounce() {
+		// TODO Auto-generated method stub
```

Comment:
Is there a reason for these comments? The methods don't look auto-generated anymore.